core/lib/multivm/src/tracers/validator/vm_latest/mod.rs

```diff
@@ -81,6 +86,56 @@ impl<H: HistoryMode> ValidationTracer<H> {
                             called_address,
                         ));
                     }
+                    // If this is a call to the timestamp asserter, extract the function arguments and store them in ValidationTraces.
+                    // These arguments are used by the mempool for transaction filtering. The call data length should be 68 bytes:
+                    // a 4-byte function selector followed by two U256 values.
+                    // TODO: if any more functions will be added to the TimestampAsserter contract in the future, verify the function selector too
+                    if Some(called_address) == self.timestamp_asserter_address
+                        && far_call_abi.memory_quasi_fat_pointer.length == 68
```

Comment:
The reason I implemented it that way - was because that's the way it's done right above for [KECCAK](https://github.com/matter-labs/zksync-era/blob/main/core/lib/multivm/src/tracers/validator/vm_latest/mod.rs#L52). 

I tried doing in the way you suggested above - calldata gets red always as zeroes. Does any modificaiton need to be done to the snippet you provided?